### PR TITLE
BETA: Register plow.is-a.dev

### PR DESCRIPTION
**Pull request by PlowDev**
Added `plow.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/clicpow.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "ClicpoW",
+        "email": "clicpowytb@gmail.com"
+    },
+    "record": {
+        "CNAME": ["https://mlkfkqd.pages.dev/"],
```
Comment (Stef-00012) [inline domains/clicpow.json:7]: Invalid CNAME record (must be a string, not an array)

Comment (Stef-00012) [reply domains/clicpow.json:7]: also CNAME can't have `https://` or `/`

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Review by wdhdev — APPROVED** [review]

File: domains/clicpow.json (hunk 1 above)
Comment (PlowDev) [reply domains/clicpow.json]: it works

Comment (CuteDog5695) [reply domains/clicpow.json]: thats because maintainer updates the file for you

Comment (PlowDev) [reply domains/clicpow.json]: yea i saw thanks

**Pull request merged**